src/gui/widgets/multi_page.cpp

```diff
@@ -154,7 +169,8 @@ void swap_grid(grid* g,
 void multi_page::finalize(const std::vector<string_map>& page_data)
 {
 	assert(generator_);
-	generator_->create_items(-1, page_builder_, page_data, nullptr);
+	//FIXME
+	generator_->create_items(-1, page_builders_.begin()->second, page_data, nullptr);
```

Comment:
That FIXME looks highly suspicious! Is the issue here that it's not even using the additional page builders? If so, that's a blocker for merging this.